lib/src/geojson.dart

```diff
@@ -35,9 +36,7 @@ abstract class GeoJSONObject {
       };
 
   static GeoJSONObject fromJson(Map<String, dynamic> json) {
-    GeoJSONObjectType decoded = json['type'] is GeoJSONObjectType
```

Comment:
All of the reformatting could cause the `dart format` tool to fail our CI checks

Reply:
Resolved in https://github.com/dartclub/turf_dart/pull/96